src/duties/browserResults.js

```diff
@@ -0,0 +1,46 @@
+let _ = require('lodash/fp')
+
+let hasBrowserErrors = ({ browserResults, fail, message }) => {
+  if (_.isNil(browserResults)) {
+    message('Could not find any browser results.')
+    return
+  }
+
+  let { result, summary } = browserResults
+
+  if (result && summary) {
+    let { error, failed, exitCode } = summary
+    if (error === false && failed === 0 && exitCode === 0) {
+      message('Your PR has no browser errors. Great job!')
+      return
+    }
+
+    if (failed > 0) {
+      _.flow(
+        // Flatten test results into one array.
+        _.reduce((flattened, other) => flattened.concat(other), []),
```

Comment:
since result in an object, you can just do a `_.values` and then a `_.concat` in the flow